### PR TITLE
docs(Contributing): Contribution warning

### PR DESCRIPTION
**Pull request by tamoore**
Warning about only posting details that are ok for the 🌏 to  👀

### CODE REVIEW

**Comment by markdalgleish** [comment]
👏  This is great! I'd even raise the prominence of this, and flesh it out a bit more, immediately starting the readme with this:

⚠️🌏👀  First and foremost, remember that this repo is **open source**.

Don't post anything or commit any code that isn't ready for the entire world to see. If you're introducing new patterns, try to keep it abstract, rather than making specific reference to the features under development. While a lot of this information is probably harmless, it's better to be safe than sorry.

**Comment by tamoore** [comment]
Did you want it added to the `README` instead? Sorry I missed that part, I amended the `CONTRIBUTING` doc. But I suppose it will get more eyes if it's in the README.

**Comment by markdalgleish** [comment]
Sorry, I misspoke, I did actually mean `CONTRIBUTING.md` 😁

**Comment by markdalgleish** [comment]
We can probably drop the leading `>`s now, that visual styling looks a bit weird now that there's a bit more text.

**Comment by markdalgleish** [comment]
🎨 👍  (Just be sure to tidy up the commit message when merging)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Contributing to seek-style-guide
 
-> ⚠️ 🌏 👀 First and foremost, remember that this repo is **open source**.
+⚠️ 🌏 👀 First and foremost, remember that this repo is **open source**.
 Don't post anything or commit any code that isn't ready for the entire world to see. If you're introducing new patterns, try to keep it abstract, rather than making specific reference to the features under development. While a lot of this information is probably harmless, it's better to be safe than sorry.
```
Comment (markdalgleish) [inline CONTRIBUTING.md]: Oh, just noticed you're missing an extra line break here 🕵

Comment (tamoore) [inline CONTRIBUTING.md]: I'm confused.com. Did you want two lines between line `4` and `5`?

Comment (markdalgleish) [inline CONTRIBUTING.md]: Yeah—the single line break isn't being honoured in the final output when reading it on GitHub.

Comment (markdalgleish) [inline CONTRIBUTING.md]: Sorry, I mean between lines 3 and 4.

**Pull request merged**